src/transformers/models/xlm/modeling_tf_xlm.py

```diff
@@ -708,6 +691,7 @@ def __init__(self, config, *inputs, **kwargs):
         output_type=TFBaseModelOutput,
         config_class=_CONFIG_FOR_DOC,
     )
+    @unpack_inputs
```

Comment:
Apologies for being picky about this one -- other PRs have been adding the decorator as the top decorator. Can you please move it, so it stays consistent with other models? 🙏  (there are more instances of this in the file)

Reply:
Sure thing. I've moved it as top decorator :)

Reply:
There are at least two more instances of `@unpack_inputs` not being the top decorator -- can you change them as well? :)

Reply:
(and don't forget the `make fixup` before committing)

Reply:
Ahh sure, apologize for this. Will add the rest and run `make fixup` before committing 👍